From 1a1570d5577086a37817ef4b5eb76527f5d0927d Mon Sep 17 00:00:00 2001
From: Jamie Mansfield <dev@jamierocks.uk>
Date: Tue, 11 Apr 2017 21:35:07 +0100
Subject: [PATCH] Fix project paths


diff --git a/creator-tools/build.gradle b/creator-tools/build.gradle
index e11a201..f548d9e 100644
--- a/creator-tools/build.gradle
+++ b/creator-tools/build.gradle
@@ -5,7 +5,7 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    compile project(':launcher-builder')
+    compile project(':Launcher:launcher-builder')
     compile 'org.eclipse.jetty:jetty-server:9.3.1.v20150714'
     compile 'com.jidesoft:jide-oss:3.6.10'
 }
diff --git a/launcher-builder/build.gradle b/launcher-builder/build.gradle
index 828447c..1581c6a 100644
--- a/launcher-builder/build.gradle
+++ b/launcher-builder/build.gradle
@@ -7,7 +7,7 @@ jar {
 }
 
 dependencies {
-    compile project(':launcher')
+    compile project(':Launcher:launcher')
     compile 'org.tukaani:xz:1.0'
     compile 'org.apache.commons:commons-compress:1.9'
 }
diff --git a/launcher-fancy/build.gradle b/launcher-fancy/build.gradle
index ef091f8..af01b33 100644
--- a/launcher-fancy/build.gradle
+++ b/launcher-fancy/build.gradle
@@ -7,7 +7,7 @@ jar {
 }
 
 dependencies {
-    compile project(':launcher')
+    compile project(':Launcher:launcher')
     compile 'com.github.insubstantial:substance:7.3'
 }
 
-- 
2.12.0

